subt_ign/launch/cave_circuit.ign

```diff
@@ -1038,6 +1040,47 @@
     "  <command>roslaunch --wait subt_ros vehicle_topics.launch world_name:=#{$worldName} name:=#{_name} uav:=#{uav} laser_scan:=#{laserScan} stereo_cam:=#{stereoCam} rgbd_cam:=#{rgbdCam}</command>\n"\
     "</executable>\n"
   end
+
+  def spawnTeamBase(_x, _y, _z, _yaw)
+    "<plugin name=\"ignition::launch::GazeboFactory\"\n"\
+    "        filename=\"libignition-launch-gazebo-factory.so\">\n"\
+    "  <name>TeamBase</name>\n"\
+    "  <allow_renaming>false</allow_renaming>\n"\
+    "  <world>#{$worldName}</world>\n"\
+    "  <is_performer>false</is_performer>\n"\
+    "  <sdf version='1.6'>\n"\
+    "  <model name='TeamBase'>\n"\
+    "    <pose>#{_x} #{_y} #{_z} 0 0 #{_yaw}</pose>\n"\
+    "    <static>true</static>\n"\
+    "    <link name='link'>\n"\
+    "      <pose>0 0 0.05 0 0 0</pose>\n"\
+    "      <visual name='visual'>\n"\
+    "        <geometry>\n"\
+    "          <box>\n"\
+    "            <size>.1 .1 .1</size>\n"\
+    "          </box>\n"\
+    "        </geometry>\n"\
+    "      </visual>\n"\
+    "    </link>\n"\
+    "    <!-- Publish robot state information -->\n"\
+    "    <plugin filename=\"libignition-gazebo-pose-publisher-system.so\"\n"\
```

Comment:
I think we don't need to use pose publisher system for this simple model and  instead we can just use a `static_transform_publisher` in `teambase_topics.launch`, which helps to reduce data published to the `/tf` topic

e.g. the `teambase_topics.launch` would then be:

```
<?xml version="1.0"?>
<launch>
  <node pkg="tf2_ros" type="static_transform_publisher" name="tf_teambase_static" args="0 0 0.05 0 0 0 TeamBase TeamBase/link"/>
</launch>
```

Reply:
Thanks, this should be fixed in 32252498f9252f310d06120ffc17673e307b2e24